feat: Parameterized linked service for Azure SQL MI pipelines (#47)
delphix/dcs-for-azure-templates: dcsazure_AzureSQL_MI_to_AzureSQL_MI_mask_pl/README.md

1. Configure the hosted metadata database and associated Azure SQL service (version `V2025.02.04.0`+).
1. Configure the DCS for Azure REST service.

> 1. Configure the AzureSQL MI linked service.
>    * It is helpful for the linked service to be parameterized with the following parameter:
>       * `LS_DATABASE` - database name in the linked service

Reviewer: ```suggestion
1. Configure the Azure SQL MI linked service.
   * Parameterize the linked service with the following parameter:
      * `LS_DATABASE` - The name of the database to connect to
```
